Core/src/test/java/org/tribuo/test/Helpers.java

```diff
@@ -184,4 +185,34 @@ public static <T extends Output<T>> void testSequenceModelSerialization(Sequence
             Assertions.fail("Failed to deserialize sequence model class " + model.getClass().toString(), ex);
         }
     }
+
+    public static boolean topFeaturesEqual(Map<String, List<Pair<String,Double>>> first, Map<String, List<Pair<String,Double>>> second, double tolerance)  {
```

Comment:
This method adds 30+ lines of code but is never used.  Probably harmless given that its in test code - but still a bit suspicious.

Reply:
Ahh, I frequently use that kind of method when I'm comparing implementations, but I kept taking it back out again and then having to remember how to re-implement it (or fake it up using watches in the debugger). So this time I just committed it. I agree it's unused most of the time which is annoying.